mautic/mautic
# [UI] Improving contrast for list groups/tabs

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [X]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR improves all list groups for more contrast.

This PR is based [on another PR](https://github.com/mautic/mautic/pull/13735), where I introduce tokens to replace most of the UI variables, aiming to improve consistency in all components. **We need it to be merged before this one becomes available for code review.**

Before:
![image](https://github.com/mautic/mautic/assets/116097999/674d9fa6-c136-464d-84b2-8f3a71c2c179)


After:
![image](https://github.com/mautic/mautic/assets/116097999/98dacb9a-4760-4639-b472-8bfc257ea5bb)





#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Roles > New > Permissions
3. Assets > New > Remote > Browse Remote Files
4. Contacts > New
5. Companies > New
6. Top navbar > your name > Account
7. Right navbar > Configuration
8. Right navbar > System info
